bedrock/mozorg/tests/test_helper_misc.py

```diff
@@ -24,15 +24,15 @@
 
 TEST_DONATE_LOCALE_LINK = {
     'default':
-        'https://sendto.mozilla.org/page/contribute/Give-Now?source={source}',
+        'https://donate.mozilla.org/en-US/?source={source}',
     'en-US':
-        'https://sendto.mozilla.org/page/contribute/givenow-seq?'
-        'preset=2&source={source}&ref=EOYFR2014&utm_campaign=EOYFR2014'
-        '&utm_source=mozilla.org&utm_medium=referral&utm_content={source}',
+        'https://donate.mozilla.org/?amount=50'
```

Comment:
We can be explicit. @flodolo also suggested that we rethink the way we build these URLs and perhaps use something more like https://github.com/mozilla/donate.mozilla.org/blob/master/data/currencies.js

Basically we should be able to inject more than just the `{source}`, ideally we should also do `{locale}` and `{currency}'. Perhaps we can also set thing like presets and defaults but, then we probably want to always set them unless, passing a parameter such as `presets=` would be fine on the donate.m.o side.